### DISCUSSION TITLE
[Art competition] FUZZER

### DISCUSSION THREAD

**Discussion by Jihwan-Yoon** [prosyslab-classroom/cs348-information-security#414] [Art competition]
![output](https://github.com/prosyslab-classroom/cs348-information-security/assets/78306242/43950799-269d-4aef-8566-2da8b035deaa)

## Short Description
For me, fuzzer means a fisherman fishing with a huge net.

## Long Description (up to 3 sentences)
Like fuzzer, which doesn't know where the bug is and finds it by covering wide range as much as possible, fishermen who fish with nets don't know where the fish is, but try to fish with the largest net as possible. And just as the area of net is not only important, but the density of the net is also important, the number of lines which fuzzer covers is not the only important.

**Comment by KihongHeo** [comment]
Nice.
